spec/requests/api/service_dialogs_spec.rb

```diff
@@ -87,6 +87,14 @@
       expect_query_result(:service_dialogs, dialogs.count, dialogs.count)
       expect_result_resources_to_include_data("resources", "label" => dialogs.pluck(:label))
     end
+
+    it "query service dialogs of a Template fetches content via the virtual attribute accessor" do
```

Comment:
Very minor nit, but could this describe the behavior (that was broken) rather than the implementation?